Traceback (most recent call last):
  File "C:\Users\kaede\anaconda3\envs\eiyl_notebook\lib\site-packages\jupyter_cache\executors\utils.py", line 56, in single_nb_execution
    record_timing=False,
  File "C:\Users\kaede\anaconda3\envs\eiyl_notebook\lib\site-packages\nbclient\client.py", line 1112, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "C:\Users\kaede\anaconda3\envs\eiyl_notebook\lib\site-packages\nbclient\util.py", line 74, in wrapped
    return just_run(coro(*args, **kwargs))
  File "C:\Users\kaede\anaconda3\envs\eiyl_notebook\lib\site-packages\nbclient\util.py", line 53, in just_run
    return loop.run_until_complete(coro)
  File "C:\Users\kaede\anaconda3\envs\eiyl_notebook\lib\asyncio\base_events.py", line 587, in run_until_complete
    return future.result()
  File "C:\Users\kaede\anaconda3\envs\eiyl_notebook\lib\site-packages\nbclient\client.py", line 554, in async_execute
    cell, index, execution_count=self.code_cells_executed + 1
  File "C:\Users\kaede\anaconda3\envs\eiyl_notebook\lib\site-packages\nbclient\client.py", line 857, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "C:\Users\kaede\anaconda3\envs\eiyl_notebook\lib\site-packages\nbclient\client.py", line 760, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
all_temperature <- all_temperature_loaded %>%
  filter(!is.na(ext_temp)) %>%
  filter(ext_temp > 2) %>%
  format_datetime() %>%
  group_by(datetime) %>%
  summarize(mean_ext = mean(ext_temp, na.rm = TRUE))
------------------

Error in as.Date.default(date, format = "%m/%d/%Y"): do not know how to convert 'date' to class "Date"
Traceback:

1. all_temperature_loaded %>% filter(!is.na(ext_temp)) %>% filter(ext_temp > 
 .     2) %>% format_datetime() %>% group_by(datetime) %>% summarize(mean_ext = mean(ext_temp, 
 .     na.rm = TRUE))
2. withVisible(eval(quote(`_fseq`(`_lhs`)), env, env))
3. eval(quote(`_fseq`(`_lhs`)), env, env)
4. eval(quote(`_fseq`(`_lhs`)), env, env)
5. `_fseq`(`_lhs`)
6. freduce(value, `_function_list`)
7. function_list[[i]](value)
8. format_datetime(.)
9. df %>% mutate(date = as.Date(date, format = "%m/%d/%Y")) %>% 
 .     mutate(datetime = as.POSIXct(date, "America/Vancouver")) %>% 
 .     mutate(datetime = datetime + hour(time) + minute(time))   # at line 2-7 of file <text>
10. withVisible(eval(quote(`_fseq`(`_lhs`)), env, env))
11. eval(quote(`_fseq`(`_lhs`)), env, env)
12. eval(quote(`_fseq`(`_lhs`)), env, env)
13. `_fseq`(`_lhs`)
14. freduce(value, `_function_list`)
15. function_list[[i]](value)
16. mutate(., date = as.Date(date, format = "%m/%d/%Y"))
17. mutate.tbl_df(., date = as.Date(date, format = "%m/%d/%Y"))
18. mutate_impl(.data, dots, caller_env())
19. as.Date(date, format = "%m/%d/%Y")
20. as.Date.default(date, format = "%m/%d/%Y")
21. stop(gettextf("do not know how to convert '%s' to class %s", 
  .     deparse(substitute(x)), dQuote("Date")), domain = NA)
ERROR: Error in as.Date.default(date, format = "%m/%d/%Y"): do not know how to convert 'date' to class "Date"


